src/Autohide.js

```diff
@@ -3,10 +3,14 @@ define(function (require, exports, module) {
 
     /* beautify preserve:start *//* eslint-disable no-multi-spaces */
     var Mustache            = brackets.getModule("thirdparty/mustache/mustache");
+    var Dialogs             = brackets.getModule("widgets/Dialogs");
 
     var OutlineManager      = require("src/OutlineManager");
     var prefs               = require("src/Preferences");
+    var Strings             = require("strings");
+
     var placeholderTemplate = require("text!templates/placeholder.html");
+    var sliderTemplate      = require("text!templates/delaySlider.html");
```

Comment:
Please rename the file to `delayDialog` or something similar (I would like to have `Dialog` in the file name).

Reply:
Fixed.